builtin/logical/pki/storage_migrations.go

```diff
@@ -150,21 +150,27 @@ func setLegacyBundleMigrationLog(ctx context.Context, s logical.Storage, lbm *le
 	return s.Put(ctx, json)
 }
 
-func getLegacyCertBundle(ctx context.Context, s logical.Storage) (*certutil.CertBundle, error) {
+func getLegacyCertBundle(ctx context.Context, s logical.Storage) (*issuerEntry, *certutil.CertBundle, error) {
```

Comment:
nit?: The migration works properly, matches with legacy behaviour, due to the iota definition of `NotAfterBehavior` the value `ErrNotAfterBehavior` comes first. Should we either be explicit within the migration perhaps of the value we want or test that ErrNotAfterBehavior. == 0?

Reply:
Wouldn't `ImportIssuer...` be the correct place to set this explicitly? Not sure what makes import on migration different than others, but yes, zero value being err was intentional :-)

Reply:
Well the others to my knowledge have the value set through either the default on the parameter definition or end-user values, this one really is just up to the iota. I don't think we can put it within importIssuer as there are so many end-users of that function.

How about just adding a check within `storage_migrations_test.go` that a migrated issuer has a value of `ErrNotAfterBehavior` set? I'd just want something to break if that iota accidently gets re-ordered.

Reply:
`importIssuer` is already opinionated though, in other forms: we automatically construct a chain (rather than giving the option to specify a manual chain at import time), and when calling e.g., `/config/ca`, we allow importing multiple issuers, and so there's really no way to control name either.

Reply:
Good points, let's go with adding it to `importIssuer`.